src/dispatch/plugins/dispatch_aws/plugin.py

```diff
@@ -70,9 +70,13 @@ def consume(self, db_session: Session, project: Project) -> None:
                         project=project, raw=signal_data, **signal_data
                     )
                 except ValidationError as e:
-                    log.exception(
+                    log.warning(
                         f"Received signal instance that does not conform to `SignalInstanceCreate` structure, skipping creation: {e}"
                     )
+                    # Add the message to be deleted even if it doesn't conform
+                    entries.append(
+                        {"Id": message["MessageId"], "ReceiptHandle": message["ReceiptHandle"]}
+                    )
                     continue
 
                 try:
```

Comment:
@kevgliss I think we may need to call rollback on the `db_session` in these exception cases as well?